apps/conv_layer/conv_layer_generator.cpp

```diff
@@ -26,8 +28,7 @@ class ConvolutionLayer : public Halide::Generator<ConvolutionLayer> {
 
         f_conv(x, y, z, n) += filter(r.x, r.y, r.z, z) * input(x + r.x, y + r.y, r.z, n);
 
-        Func f_ReLU("ReLU");
-        f_ReLU(x, y, z, n) = max(0, f_conv(x, y, z, n));
```

Comment:
"relu" is a meaningful name in this context that we should preserve in some way.

Reply:
Fixed.